Makefile

```diff
@@ -1,11 +1,12 @@
-PREFIX=/usr/local
-BINDIR=$(DESTDIR)$(PREFIX)/bin
-MANDIR=$(DESTDIR)$(PREFIX)/share/man/man1
-DOCDIR=$(DESTDIR)$(PREFIX)/share/doc/googler
-UNAME_S:=$(shell uname -s)
+PREFIX ?= /usr/local
+BINDIR = $(DESTDIR)$(PREFIX)/bin
+MANDIR = $(DESTDIR)$(PREFIX)/share/man/man1
+DOCDIR = $(DESTDIR)$(PREFIX)/share/doc/googler
+BASHCOMPDIR = $(DESTDIR)$(PREFIX)/etc/bash_completion.d
+FISHCOMPDIR = $(DESTDIR)$(PREFIX)/share/fish/vendor_completions.d
+ZSHCOMPDIR = $(DESTDIR)$(PREFIX)/share/zsh/site-functions
```

Comment:
That's the correct behavior (or at least, one correct behavior). See [GNU Coding Standards on `sysconfdir`](https://www.gnu.org/prep/standards/html_node/Directory-Variables.html) or [FHS](https://www.linuxbase.org/betaspecs/fhs/fhs/ch04s09.html).